public/walkthroughs/walkthrough-2/walkthrough.adoc

```diff
@@ -0,0 +1,430 @@
+. In the *Staging Public Base URL*, enter:
++
+[subs="attributes+"]
+----
+https://wt2-{user-sanitized-username}-3scale.{openshift-app-host}
+----
++
+Note that this route should point to the shared staging APIcast in the *3scale* project in OpenShift.  Your administrator should have created this route for you. If it does not exist, contact your administrator to create the route.
+
+. Select *Update & test in Staging Environment*
+
+[type=verification]
+Check that the API service is available.
+You might encounter a *403: Authentication failed* message. You can ignore this message, the issue is resolved in a later step.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
+=== 3.3 Setting Fuse Aggregation App Endpoint Limits
+
+. Create a new *Application Plan*:
+.. Select *Application Plans* from the left menu.
+.. Select *Create Application Plan*.
+.. Enter the following for *Name* and *System name*:
++
+[subs="attributes+"]
+----
+{fuse-aggregator-app-name}
+----
+.. Select *Create Application Plan*.
+.. Select the *Publish* button to publish the Plan.
+
+. Select the *{fuse-aggregator-app-name}* plan to return to the edit screen.
+
+. Set a limit of 5 calls per hour:
+.. On the *Metrics, Methods, Limits & Pricing Rules* section, select the *Limits (0)* button.
+.. Select the *New usage limit* button.
+.. Set the *Period* to *hour*.
+.. Set the *Max. value* to *5*.
+.. Select *Create usage limit*.
+
+. Create a new *Application* for the *Developer* Group, assigned to the Plan:
+.. Select the *Developers* menu at the top.
+.. Select the *Developer* Account to open the *Account Summary* page.
+.. Select the *(num) Application* item from the breadcrumb to view Applications.
+.. Select the *Create Application* button in the top right.
+.. Select the *{fuse-aggregator-app-name}* Plan in the *Application plan* dropdown.
+.. Enter the following for *Name* and *Description*:
++
+[subs="attributes+"]
+----
+{fuse-aggregator-app-name}
+----
+.. Select *Create Application*.
+
+. Set a custom *User Key* for the application:
+.. On the *{fuse-aggregator-app-name}* application screen, scroll to the *API Credentials* section.
+.. Select *Set Custom Key*.
+.. In the *Custom User Key* field, enter:
++
+[subs="attributes+"]
+----
+{fuse-aggregator-app-name}
+----
+.. Select *Set Custom Key*.
+
+[type=verification]
+****
+. Select the *APIs* menu item at the top.
+
+. Expand the *{fuse-aggregator-app-name}* service.
+
+. Select the *Configure APIcast* button.
+
+. Select the *Update & test in Staging Environment* button at the bottom again.
+
+. Check that a success message is displayed, and a green line along the left side of the page.
+****
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
```

Comment:
@pb82  think you're missing docs/modules/ROOT/pages/_partials/fuse-aggregation-app-endpoint-activedocs.adoc here

Reply:
ie in prod, it looks like https://i.imgur.com/wPiP74w.png

Reply:
you're right, adding now